### DISCUSSION TITLE
user.name and user.email seems like can frame an employee or someone else to be blamed as the bad contributor. STILL BUG or FEATURE?

### DISCUSSION THREAD

**Discussion by omega13-engr** [community/community#146113] [Other Feature Feedback, Questions, & Ideas]
My colleague just reported an issue a while ago with some unusual commit history, where their personal GitHub account was displayed as the committer instead of the org account due to the git config `user.name` and `user.email` settings. Yet I know this is common from other discussions.

To investigate, I tested this by changing my `user.name` and `user.email` to another identity while using my organization’s SSH key. I observed a similar issue plus the profile picture, where the commit metadata did not align with the SSH key configuration. It's also confusing that the commit status will reflect verified and unverified to a certain contributor.

```
Commit history:

Bad commit make me getting fired...
contributor: johndoe-org <johndoe@org.org>                        abcdef3

Celebrity commit ...
contributor: github <github@github.com>              <Verified>   abcdef2

Personal commit ...
contributor: slaee <slaee@personal.com>              <Unverified> abcdef1

Update ...
contributor: slaee-org <slaee@org.org>                            abcdef0
```

**Comment by tuves** [comment]
Welcome to the GitHub Community, @slaee, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
